src/frontend/src/pages/TeamsPage/TeamSummary.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, Collapse, Col, Container, Row } from 'react-bootstrap';
+import { Team } from 'shared';
+import { useTheme } from '../../hooks/Theme.hooks';
+import { routes } from '../../utils/Routes';
+import { fullNamePipe, wbsPipe } from '../../utils/Pipes';
+import styles from '../../stylesheets/pages/ProjectDetailPage/WorkPackageSummary.module.scss';
+
+interface TeamSummaryProps {
+  team: Team;
+}
+
+const TeamSummary: React.FC<TeamSummaryProps> = ({ team }) => {
+  const [open, setOpen] = useState(false);
+  const theme = useTheme();
+
+  const membersList = (
+    <ul>
+      {team.members.map((member, idx) => (
+        <li key={idx}>{fullNamePipe(member)}</li>
+      ))}
+    </ul>
+  );
+
+  const projectsList = (
+    <ul>
+      {team.projects.map((project, idx) => (
+        <li key={idx}>
+          <Link to={`${routes.PROJECTS}/${wbsPipe(project.wbsNum)}`}>{project.name}</Link>
+        </li>
+      ))}
+    </ul>
+  );
+
+  return (
+    <Card bg={theme.cardBg} border={theme.cardBorder}>
+      <Card.Header className={styles.header} onClick={() => setOpen(!open)} aria-expanded={open}>
```

Comment:
I think name on the left, # members on the right, maybe both # members and # projects on the right, would be a nice preview

Reply:
thats a cool idea